src/index.html

```diff
@@ -2,13 +2,71 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name = "viewport"
+      content="width=device-width,
+      initial-scale=1.0"
+      >
```

Comment:
The arrows must be on the same line. Check it out everywhere
```suggestion
    <meta
      name = "viewport"
      content="width=device-width,
      initial-scale=1.0"
    >
```

Reply:
![photo_2023-05-04_17-52-57](https://user-images.githubusercontent.com/131815205/236245790-3c03a6ff-3f74-4366-b584-0a6eb10ad0e0.jpg)
![photo_2023-05-04_17-53-10](https://user-images.githubusercontent.com/131815205/236245869-56e63e9b-f0c7-47ae-b33c-52e95b3f1e73.jpg)
I checked, i have 0 errors